tests/dataset_readers/transformer_squad_test.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from allennlp.common import Params
+from allennlp.common.util import ensure_list
+
+from allennlp_rc.dataset_readers import TransformerSquadReader
+from tests import FIXTURES_ROOT
+
+
+class TestTransformerSquadReader:
+    def test_read_from_file(self):
+        reader = TransformerSquadReader()
+        instances = ensure_list(reader.read(FIXTURES_ROOT / "data" / "squad.json"))
+        assert len(instances) == 5
+
+        token_text = [t.text for t in instances[0].fields["question"].tokens]
+        token_type_ids = [t.type_id for t in instances[0].fields["question"].tokens]
+
+        assert token_text[:3] == ["[CLS]", "To", "whom"]
+        assert token_type_ids[:3] == [1, 1, 1]
+
+        assert token_text[-3:] == ["Mary", ".", "[SEP]"]
+        assert token_type_ids[-3:] == [0, 0, 1]
+
+        assert token_text[instances[0].fields["context_span"].span_start] == "Architectural"
+        assert token_type_ids[instances[0].fields["context_span"].span_start] == 0
+
+        assert token_text[instances[0].fields["context_span"].span_end + 1] == "[SEP]"
+        assert token_type_ids[instances[0].fields["context_span"].span_end + 1] == 1
+        assert token_text[instances[0].fields["context_span"].span_end] == "."
+        assert token_type_ids[instances[0].fields["context_span"].span_end] == 0
+
+        assert token_text[
+            instances[0]
+            .fields["answer_span"]
+            .span_start : instances[0]
+            .fields["answer_span"]
+            .span_end
+            + 1
+        ] == ["Saint", "Bern", "##ade", "##tte", "So", "##ubi", "##rous"]
+
+    def test_can_build_from_params(self):
```

Comment:
I don't think we need this kind of test anymore.

Reply:
It's gone. 12affcae7852e06dbde4b4586c65138d7d89c59a